content/posts/irods-development-update-august-2024.md

```diff
@@ -0,0 +1,260 @@
+Title: iRODS Development Update: August 2024
+Date: 2024-08-14 17:00
+Author: Kory Draughn
+Slug: irods-development-update-august-2024
+Status: published
+
+
+Hello reader,
+
+iRODS 4.3.3 is very close. We expect we'll have a release in a matter of days/weeks.
+
+Starting with the server, it was discovered that the `rc_replica_truncate` API was not returning information in the JSON output parameter as documented. That has been resolved. We've confirmed the server works with MariaDB. For this to work, the MySQL ODBC Connector must be used. Work on the database plugin project template is progressing well. When it is complete, users looking to add support for new databases will be able to jump right into writing code instead of spending time configuring CMake. For users running iRODS on CentOS 7, the Consortium will not be releasing iRODS 4.3.3 packages for this platform.
+
+On the packaging side of things, plugins and clients provided by the Consortium will now include the version of iRODS used to compile the binaries. For example, the Logical Quotas rule engine plugin for iRODS 4.3.3 / Ubuntu 22.04 will have a package name **irods-rule-engine-plugin-logical-quotas_4.3.3.0-0+4.3.3~jammy_amd64.deb**. This is the first step towards independent versioning for plugins and clients. We'll have more to share as things develop.
+
+The PAM Interactive Authentication plugin has received a good number of improvements since last month. Issues with TLS/SSL and account spoofing _(thanks Leiden University)_ have been resolved. The plugin now uses configuration from the **R_GRID_CONFIGURATION** table just like the PAM Password Authentication plugin. And, we've successfully demonstrated PAM Password and Kerberos flows using this plugin. We're very close to a first release so stay tuned.
+
+Users of the Python iRODS client (PRC) will be happy to know that 2.1.0 has been released. To learn what's changed since 2.0.0, see the release notes at [https://github.com/irods/python-irodsclient/releases/tag/v2.1.0](https://github.com/irods/python-irodsclient/releases/tag/v2.1.0).
```

Comment:
```suggestion
Users of the Python iRODS client (PRC) will be happy to know that 2.1.0 has been released. To learn what's changed since 2.0.1, see the release notes at [https://github.com/irods/python-irodsclient/releases/tag/v2.1.0](https://github.com/irods/python-irodsclient/releases/tag/v2.1.0).
```

Reply:
Done.